examples/references-count/README.md

```diff
@@ -0,0 +1,17 @@
+# Entry List location example
+This example shows how to build an advanced app with an `EntryList` location. It counts a number of references for the entries.
+
+## How to use
+
+[![Install to Contentful](https://www.ctfstatic.com/button/install-small.svg)](https://app.contentful.com/deeplink?link=apps&id=50jZ722cWV7JOVj4SW10jI)
+
```

Comment:
Does that make sense in the examples folder? People will probably create their own app out of this